new.go

```diff
@@ -278,14 +277,16 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 		}
 	}
 
-	coptions := storage.ContainerOptions{}
-	coptions.IDMappingOptions = newContainerIDMappingOptions(options.IDMappingOptions)
-
+	coptions := storage.ContainerOptions{
+		LabelOpts:        options.CommonBuildOpts.LabelOpts,
+		IDMappingOptions: newContainerIDMappingOptions(options.IDMappingOptions),
+	}
```

Comment:
Might as well move this definition into the loop to have it create a new `ContainerOptions` for each call to `CreateContainer()`.